README.md

```diff
@@ -130,9 +130,26 @@ spec:
     key: token
 ```
 
-This will fetch the secret, and get the value in `token` and use that to
-authenticate the API call to GitHub.
-
+This will fetch the secret, and get the value in `token` and use that to authenticate the API call to GitHub. The secret may contain multiple values. For example: 
```

Comment:
I think we could maybe clarify that the `key` is looked up in the data for the `Secret` what do you think?

Reply:
Yes. Please make the required changes.

Reply:
Should I remove the example?

Reply:
I think the example is useful, I used `token` which was the default, which didn't clarify it, so adding another example should hopefully make it easier :-)